logstash-core/lib/logstash/util/duration_formatter.rb

```diff
@@ -1,15 +0,0 @@
-# encoding: utf-8
-require "chronic_duration"
-module LogStash::Util::DurationFormatter
-  CHRONIC_OPTIONS = { :format => :short }
-
-  # Take a duration in milliseconds and transform it into
-  # a format that a human can understand. This is currently used by
-  # the API.
-  #
-  # @param [Integer] Duration in milliseconds
-  # @return [String] Duration in human format
-  def self.human_format(duration)
-    ChronicDuration.output(duration / 1000, CHRONIC_OPTIONS)
```

Comment:
This is the only reference to `ChronicDuration` either here or in the logstash-plugins org, so we should be able to also remove the `chronic_duration` dependency from `lib/logstash-core/logstash-core.gemspec:60`